decompy/DataGathering/RepoFilter.py

```diff
@@ -1,7 +1,10 @@
 import json
+import requests

```

Comment:
Why :-/ We are using requests and urllib in THE SAME FILE. It's fine for now, but using two libraries that literally do the same thing is redundant and bad coding.